examples/04.MuonScan/CosmicMuonsFromCircle.rml

```diff
@@ -29,8 +29,9 @@
                    position="(0,100,0)mm" size="(400,0,0)mm"
                    rotationAngle="90deg" rotationAxis="(1,0,0)">
             <source particle="mu-" excitedLevel="0.0" fullChain="on">
-                <energy type="TH1D" file="Muons.root" name="cosmicmuon"/>
```

Comment:
Instead of replacing I would add the new generator as a new example. The TH1D generator can still be useful, and it is good to have an example for both.

Reply:
I reverted the old examples to still use the same generator and added a new example.